testsuite/cluster-test/src/cluster_swarm/libra_node_spec_template.yaml

```diff
@@ -37,19 +24,21 @@ spec:
     - mountPath: /opt/libra/data
       name: data
   - name: debug
-    image: 853397791086.dkr.ecr.us-west-2.amazonaws.com/libra_validator:{image_tag}
+    image: 853397791086.dkr.ecr.us-west-2.amazonaws.com/libra_e2e:{image_tag}
```

Comment:
I think you will have to update `docker/build-aws.sh`'s `--build-all-cti` list of projects to include the `libra_e2e` image. According to our ECR, that repo hasn't been pushed to in more than a year

Reply:
er nevermind

Reply:
This image gets built in the validator CodeBuild spec right?

Reply:
So actually libra_e2e is build and pushed as part of build and push of libra_validator

Reply:
Got it!